.github/workflows/acceptance-tests-runner.yml

```diff
@@ -526,8 +536,9 @@ jobs:
             ./internal/service/maintenancewindow
         run: make testacc
   autogen:
-    needs: [ change-detection, get-provider-version ]
-    if: ${{ needs.change-detection.outputs.autogen == 'true' || inputs.test_group == 'autogen' }}
+    needs: [change-detection, get-provider-version]
+    # autogen doesn't support SA yet
+    if: ${{ inputs.use_sa == false && (needs.change-detection.outputs.autogen == 'true' || inputs.test_group == 'autogen') }}
```

Comment:
q: Why are we stating SA are not supported for autogen? Curious if we found a limitation or if we simply want to exclude this test group when running with SA.